django/conf/__init__.py

```diff
@@ -105,6 +105,21 @@ def __init__(self, settings_module):
                 setattr(self, setting, setting_value)
                 self._explicit_settings.add(setting)
 
+        if self.SECRET_KEY_FILE and not self.SECRET_KEY:
```

Comment:
If I already have `SECRET_KEY` and I want to migrate to use `SECRET_KEY_FILE`, this won't work, right? Raising a warning or perhaps `ImproperlyConfigured` would be good.